<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCraftFixture</class>
 <widget class="QDialog" name="DialogCraftFixture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>请选择工装</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFixtureType">
       <property name="text">
        <string>工装类型：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxFixtureType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFixtureSerial">
       <property name="text">
        <string>工装序号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditFixtureSerial"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAccept">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReject">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
